src/main/java/bruno/UI.java

```diff
@@ -0,0 +1,42 @@
+package bruno;
+
+/**
+ * The UI class is responsible for the display of messages, i.e, it controls the user-interface of the task
+ * management system.
+ */
+public class UI {
+
+    /**
+     * Displays the divider lines before and after each command.
+     */
+    public void displayLines() {
+        for (int i = 0; i < 20; i++) {
+            System.out.print((i == 0 ? "-" : " -"));
+        }
+        System.out.println();
+    }
+
+    /**
+     * Displays the greeting at the start of the program.
+     */
+    public String displayGreeting() {
+        String name = "Bruno";
+        return "Woof Woof! I'm " + name + " 🐾" + "\nHow can I help you?";
```

Comment:
Cute emotion! I'm not sure if this is a standard but I have this habit of putting "\n" at the end of the line instead of at the start of next line. Even if both are same effect I guess 😆 
```suggestion
        return "Woof Woof! I'm " + name + " 🐾 \n" + "How can I help you?";
```